development/readme_eclipse_4.32.html

```diff
@@ -279,7 +279,7 @@ <h4>
   <ul>
     <li><a
       class="mozTocH4"
-      name="mozTocId693657">As shown </a><a href="#TargetOperatingEnvironments">above</a>, Eclipse 4.32 requires at least a Java SE 17. Perhaps an older version of the VM is being found in your path. To
+      name="mozTocId693657">As shown </a><a href="#TargetOperatingEnvironments">above</a>, Eclipse 4.32 requires at least a Java SE 21. Perhaps an older version of the VM is being found in your path. To
```

Comment:
This has to stay Java SE 17 as this document is for Eclipse SDK. One can even install a plugin that requires Java 22(even 23) now but that is plugin requirement much like what we see in EPP packages.
Another possibility is to reword it like "requires at least a Java SE 17 but Java SE 21 for greater compatibility with plugins that already have higher requirements".

Reply:
I think I like your alternative wording best, what do you think about the new version?

Reply:
Sounds good.